client/src/game/ui/gameCommands.ts

```diff
@@ -289,7 +289,7 @@ commands.set('init', (data: InitData) => {
   const metadata: GameMetadata = {
     options: data.options,
     ourPlayerIndex: data.seat,
-    spectating: data.spectating,
+    spectating: data.spectating || data.replay || data.sharedReplay,
```

Comment:
should just be data.spectating || data.replay

Reply:
works for me